tests/user.js

```diff
@@ -24,3 +25,18 @@ test('should parse token in ctor', (t) => {
 
   t.is(user.token.expires_at, 1000000);
 });
+
+test('should not log token on error', (t) => {
```

Comment:
```suggestion
test.only('should not log token on error', (t) => {
```

Since we're patching a global object (`console`).